src/routes/(app)/pos/touch/+page.server.ts

```diff
@@ -0,0 +1 @@
+export const load = async ({}) => {};
```

Comment:
Should add `layoutReset: true` in the data, to avoid rendering all the html for the regular layout (even if it's hidden)